lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -4430,6 +4430,30 @@ LogicalResult ConvertAtenOp<AtenConstantPadNdOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenCatOp>::matchAndRewrite(
+    AtenCatOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+  int64_t dim;
+  if (!matchPattern(op.getDim(), m_TorchConstantInt(&dim)))
```

Comment:
can check if the dim is >= 0 since tosa doesn't support signed number for the dim argument?

Reply:
just add support for negative dim